tb_plugin/fe/src/app.tsx

```diff
@@ -122,12 +125,18 @@ const useStyles = makeStyles((theme) => ({
   },
   iconButton: {
     padding: '8px'
-  }
+  },
+  container: {
+    display: 'flex',
+    padding: '80px',
```

Comment:
Any reason for the enormous padding?  If you want the text closer to the center of the screen, I'd prefer a margin so the card to text distance is consistent with other cards.